gulpfile.js

```diff
@@ -235,15 +235,21 @@ gulp.task('run-unit-tests', function (cb) {
 });
 
 gulp.task('run-test-integration', function() {
-  return gulp.src('./tests/{common,integration}/**/*_test.js', { read: false }).pipe(mocha());
+  return gulp.src(
```

Comment:
__question:__ What do you think if we align such cases like this?
```
return gulp.src(
  TESTS_ROOT + '{common,integration}/**/*_test.js', { read: false }
)
.pipe(mocha());
```
Don't have strong opinion, just asking :)